Documentation improvements and general typo fixes. (#340)
michael-simons/neo4j-migrations: docs/usage.adoc

== Common operations

[[usage_common_clean]]
=== Clean

`clean` applies by default to the schema database.
It will remove Neo4j-Migrations related nodes and relationships.
If there is no schema database selected, it works on the optional target database.
If this isn't configured either, the users home database will be used.

The clean operation will search for

* Migration chains (those are the nodes containing information about the applied migrations)
* Any log from this Neo4j-Migrations
* Any constraints created by Neo4j-Migrations

and will delete and drop them in that order.
This is a *destructive* operation, so make sure not to apply it to your production database without thinking at least twice.
It cannot be undone via Neo4j-Migrations.

The operation takes in a boolean parameter.
When set to `false`, only the <<concepts_chain,migration chain>> for the currently configured target database will be deleted.
When set to `true`, all objects created by Neo4j-Migrations will be deleted.

[[usage_common_info]]
=== Info

The `info` operations returns information about the context, the database, all applied and all pending applications.

[[usage_common_migrate]]
=== Migrate / apply

> The `migrate` command (or `apply` in case of the Java-based migration) does exactly that:

Reviewer: ```suggestion
The `migrate` command (or its underlying method `apply` in the Migrations Core API) does exactly that:
```
